internal/cf/client.go

```diff
@@ -83,6 +90,12 @@ func newOrganizationClient(organizationName string, url string, username string,
 	if err != nil {
 		return nil, err
 	}
+	// isResourceCacheEnabled, _ := strconv.ParseBool(os.Getenv("RESOURCE_CACHE_ENABLED"))
```

Comment:
You mentioned that this is part of another PR.
Therefore, just remove it from here?